File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.js
function solution(emergency) {
    var answer = [];
    let sorted = [...emergency].sort((a, b) => b - a);
    for (let i = 0; i < emergency.length; i++) {
        let rank = sorted.indexOf(emergency[i]) + 1;
        answer.push(rank);
    }
    return answer;
}

File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.py
def solution(emergency):
    arr = sorted(emergency, reverse = True)
    answer = []
    for i in emergency:
        idx = arr.index(i)+1
        answer.append(idx)
    return answer
    
    # answer = []
    # for i in emergency:
    #     index = 1
    #     for j in emergency:
    #         if i < j:
    #             index += 1
    #     answer.append(index)
    # return answer